.github/workflows/changelog.yml

```diff
@@ -0,0 +1,19 @@
+name: "Changelog Check"
+
+on:
+  pull_request:
+    types:
+      - "opened"
+      - "synchronize"
+      - "reopened"
+      - "edited"
+    issue_comment:
```

Comment:
Does this trigger the workflow on every issue comment in this repository?

Reply:
it does. actually the `edited` type of the pull_request should be enough. 
Although it will also trigger it if the title is edited. But I think we can live with that for now